lua/refactoring/code_generation/python.lua

```diff
@@ -1,8 +1,26 @@
 local utils = require("refactoring.code_generation.utils")
 
+local function get_prefix(start_col)
```

Comment:
I like where this is going, but perhaps we can make it a bit more generic anmed and move it to utils.

// could take in whitespace type (tabs vs spaces) or something else within make whitespace.
`make_whitespace(count, opts = {}) ...`